specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/stable/2020-06-25/guestconfiguration.json

```diff
@@ -802,16 +802,28 @@
           "type": "string",
           "description": "Version of the guest configuration."
         },
-        "contentUri": {
-          "readOnly": true,
+        "contentUri": {          
           "type": "string",
           "description": "Uri of the storage where guest configuration package is uploaded."
         },
-        "contentHash": {
-          "readOnly": true,
+        "contentHash": {          
           "type": "string",
           "description": "Combined hash of the guest configuration package and configuration parameters."
         },
+		"assignmentType": {
+			"type": "string",
+			"description": "Specifies the assignment type and execution of the configuration. Possible values are Audit, DeployAndAutoCorrect, ApplyAndAutoCorrect and ApplyAndMonitor.",
+			"x-ms-enum": {
+            "name": "assignmentType",
+            "modelAsString": true
+          },
+          "enum": [
```

Comment:
Sorry. One more question, I just want to double confirm. Seems the possible values of assignmentType are different with configurationMode's. Is it expected?

Reply:
yes that is correct

Reply:
yes that is correct